### DISCUSSION TITLE
RPM build/repository

### DISCUSSION THREAD

**Discussion by theAeon** [Audio4Linux/JDSP4Linux#38] [Packages]
Hey all-just spent a decent bit of time getting the pipewire flavor running on fedora's copr infrastructure and it seems to be working well on fedora 35 (and building on 34/rawhide/suse tumbleweed). I had to flatten the submodules because, even though rpkgv3 can handle it fine, fedora's copr is stuck on rpkgv2 for the time being. This, well, makes it a bit hard to pull request. In the meantime, I'll maintain it on my [repo](https://github.com/theAeon/JDSP4Linux) in one way or another. 2.2 is currently on the rpm branch if anyone wants to help out with implementing pulse/finds bugs/things i messed up/whatever. The repo is available at https://copr.fedorainfracloud.org/coprs/arrobbins/JDSP4Linux/ and installation can be done from yum as with any copr repository. Let me know!

**Comment by timschneeb** [comment]
Thank you! I've added credits and instructions to the readme. Feel free to send a PR if anything changes.

**Comment by theAeon** [comment]
update: it looks like COPR is building submodules now. Should be able to integrate it here.

**Reply by theAeon** [reply]
actually its just flat broken using rpkg generally

guess i need to figure out how to convert it to tito
